### PR TITLE
chore: add message to aws lambda when no body is provided

### PR DESCRIPTION
**Pull request by ysael**

### CODE REVIEW

File: packages/aws-lambda/src/r5/handler.ts [hunk 1]
```diff
@@ -122,6 +122,9 @@ export function fhirSubscriptionHandler(
     if (!event.body) {
       return {
         statusCode: 204,
+        body: JSON.stringify({
+          message: `Body was not provided, noop for the following ${subscription}`,
```
Comment (julienblin) [inline packages/aws-lambda/src/r5/handler.ts]: Did you test it? My guess is that this print `Unable to find a subscription for [object Object]`.

**Review by julienblin — CHANGES_REQUESTED** [review]
Thank you for the PR.

A couple of things:
- Message is probably wrong - not sure if you took the time to test it.
- This is missing a changeset

File: packages/aws-lambda/src/r5/handler.ts [hunk 2]
```diff
@@ -111,7 +111,9 @@ export function fhirSubscriptionHandler(
       return {
         statusCode: 404,
         body: JSON.stringify({
-          message: `Unable to find a subscription for ${subscription}`,
+          message: `Unable to find a subscription for ${JSON.stringify(
```
Comment (julienblin) [inline packages/aws-lambda/src/r5/handler.ts]: Ouch - are you sure we are not leaking the Authorization header by doing this?
Also - there should be a change in the r4b directory. This package has not been built.

**Review by julienblin — CHANGES_REQUESTED** [review]
I believe this contains a security problem.

File: packages/aws-lambda/src/r5/handler.ts (hunk 2 above)
Comment (ysael) [reply packages/aws-lambda/src/r5/handler.ts]: that code was already there was it intentional to log [Object] here? I will remove for both

**Review by julienblin — APPROVED** [review]
LGTM!
Thank you!

**Pull request merged**